packages/renderer/src/validate-fps-for-gif.ts

```diff
@@ -0,0 +1,9 @@
+import {Codec} from 'remotion';
+
+export const validateFpsForGif = (codec: Codec, fps: number) => {
+	if (codec === 'gif' && (fps > 50 || fps < 1)) {
```

Comment:
Unused and should apply after the recalculated GIF (after `everyNthFrame`) is taken into account